api/v1alpha4/metal3machine_types.go

```diff
@@ -62,6 +62,13 @@ type Metal3MachineSpec struct {
 	// NetworkData is an object storing the reference to the secret containing the
 	// network data given by the user.
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
+
+	// When set to disabled, automated cleaning will be skipped
+	// during provisioning and deprovisioning.
+	// +optional
```

Comment:
Why do we need this tag if we have `omitempty` two lines below?

Reply:
My bad, forgot to clean that.